### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [AllayMC/Allay#225] [Giscus]
# index

https://docs.allaymc.org/

**Comment by Liyou-Fire** [comment]
世界怎么配置，能不能写个教程

**Reply by smartcmd** [reply]
See https://docs.allaymc.org//getting-started/use-terra/

**Comment by NewmoomCat** [comment]
能不能搞个玩家的API类，不然移植不了某些插件

**Reply by IWareQ** [reply]
https://github.com/AllayMC/Allay/blob/master/api/src/main/java/org/allaymc/api/entity/interfaces/EntityPlayer.java

**Comment by NewmoomCat** [comment]
谢谢
